package.json

```diff
@@ -105,6 +105,8 @@
   },
   "scripts": {
     "test": "NODE_ENV=test nyc --reporter=text mocha --exit --opts test/mocha.opts",
+    "check": "eslint --ignore-path .gitignore --ignore-pattern node_modules --quiet .",
```

Comment:
This is the other change of significance.

Reply:
I'm surprised that that wasn't in there already